tutorials/sim/plot_07_SimMulti.py

```diff
@@ -86,72 +130,55 @@
 ]
 
 # Simulate a set of signals
-sims_across_params = sim_across_values(sim_powerlaw, multi_params, 3)
+sims_across_params = sim_across_values(sim_powerlaw, multi_params)
 
 ###################################################################################################
 #
-# The output of the above is a :class:~.MultiSimulations object that stores sets of simulations
-# across different parameters, and relevant metadata. Each set of simulations is stored within
-# this object as a :class:~.Simulations object.
+# The output of the above is a :class:~.VariableSimulations object that stores simulations
+# across varying simulation parameters (same as with the sampled simulations).
 #
 
 ###################################################################################################
 
-# The length of the object is the number of parameter sets
-print('# of sets of signals:', len(sims_across_params))
+plot_multi_time_series(None, sims_across_params)
```

Comment:
Yeh, I tried to keep the same API as `plot_time_series` (and other time series plots), so the first input is `times`, but it's optional if you want to pass None and plot in terms of samples. I don't really want to make this function inconsistent with the other, nor change the current API for all of them. 

Since this specific occurence is in the tutorials, I think it's better practice to explicitly create and use a times definition, so I've edited this file to do so